src/status_im/chat/commands/core.cljs

```diff
@@ -0,0 +1,139 @@
+(ns status-im.chat.commands.core
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.icons.vector-icons :as vector-icons]
+            [status-im.ui.components.colors :as colors]
+            [status-im.i18n :as i18n]
+            [status-im.chat.styles.message.message :as style]))
+
+(defprotocol Command
+  "Protocol for defining command message behaviour"
+  (id [this] "Identifier of the command, used to look-up command display name as well")
```

Comment:
Is that unique across all commands?

Reply:
Not really, combination of the `[id scope]` is unique - for example `/sent` command has two versions, one for 1-1 chats, another (with one extra argument) for group chats. 
Maybe `name` instead of `id` would be better, but I don't like the name-clash with core `name` function.

Reply:
👍